### DISCUSSION TITLE
数据库功能增强

### DISCUSSION THREAD

**Discussion by sysfox** [Qexo/Qexo#314] [Ideas]
在#308中，你提到
> 建议部署两个

而Qexo部署时默认mongodb的数据库名为django，建议支持自定义，以充分利用一个mongodb数据库

**Comment by sysfox** [comment]
淦，忘看文档了，请忽略这个discussion

**Comment by am-abudu** [comment]
可以使用PlanetScale/Supabase数据库部署

**Reply by GriMu** [reply]
PlanetScale免费版如果一段时间没有写入，好像数据库就会进入sleep状态，这个有办法处理吗- -

**Reply by am-abudu** [reply]
> PlanetScale免费版如果一段时间没有写入，好像数据库就会进入sleep状态，这个有办法处理吗- -

我建议使用Bit.io或者Vercel正在Beta的数据库

**Reply by am-abudu** [reply]
Vercel的数据库(选择PostgreSql)在绑定了Qexo项目后会被自动识别并使用(Redeploy后)

**Reply by GriMu** [reply]
问下，我有自己的mysql库，但是配置参数后，部署失败了，提示ssl不支持，vercel连接自己服务器的mysql不能用ip端口，只能ssl连接吗

**Reply by am-abudu** [reply]
我强烈不建议为Vercel使用自己的数据库，出现的问题你极难自行解决。请使用文档中推荐的数据库/Vercel自带数据库或者考虑本地部署

**Comment by Fgaoxing** [comment]
没有Vercel，也会休眠



在 2023-05-25 21:16:51，"GriMu" ***@***.***> 写道：

PlanetScale免费版如果一段时间没有写入，好像数据库就会进入sleep状态，这个有办法处理吗- -
